app/src/main/java/com/example/proptit_protify/data/RetrofitHelper.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.proptit_protify.data
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitHelper {
+    private val BASE_URL="https://deezerdevs-deezer.p.rapidapi.com/"
+    fun getInstance(): Retrofit{
```

Comment:
NO! getInstance() is being used when we use Singelton. Are you sure what are you doing is normal?